MetaMask/metamask-extension
# Add snaps search

Fixes #13943 

Explanation: This PR adds snaps to the settings search. `SettingsSearchList` was also updated to use font-awesome instead of including assets in the build process. 

**Note:** This settings search code needs to be refactored and snaps view search needs to be dynamically added as snaps are added/removed -- all which will be coming in a future PR.

Screenshot:
<img width="354" alt="Screen Shot 2022-03-21 at 9 44 33 PM" src="https://user-images.githubusercontent.com/41640681/159390706-f1d4e21b-3936-4053-83b8-92d30c0b2dde.png">

Testing:
- Run a flask build using `yarn start --build-type flask`
- Load the extension into chrome
- Observe changes
